### PR TITLE
Initial refactor 

### PR DESCRIPTION
**Pull request by vyasworks**
Single symptom component
Q1 and Q2 flow.
Validation on continue button

### CODE REVIEW

File: apps/web/src/components/Checkboxes.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+import { FormFieldProps } from './Radio';
+
+function Checkboxes(props: FormFieldProps) {
```
Comment (hitankar-freshworks) [inline apps/web/src/components/Checkboxes.tsx:5]: generally consistent to use array function

File: apps/web/src/components/Checkboxes.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const { label, name, options, ...rest } = props;
+  return (
+    <div>
+      <label>{label}</label>
+      <Field name={name}>
```
Comment (hitankar-freshworks) [inline apps/web/src/components/Checkboxes.tsx:10]: I think this approach is unnecessary. Why do we need to wrap checkboxes with Field component? I rather do Field type="checkbox" approach

File: apps/web/src/components/Radio.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <Field name={name}>
+        {formik => {
+          const { field } = formik;
+          return options.map(option => {
+            return (
+              <div
+                className='flex space-x-5 items-center cursor-pointer leading-none py-3'
+                key={option.key}
+              >
+                <input
```
Comment (hitankar-freshworks) [inline apps/web/src/components/Radio.tsx:34]: read my previous concern

File: apps/web/src/forms/SymptomChecker.tsx [hunk 4]
```diff
@@ -63,65 +58,108 @@ export const SymptomChecker = () => {
...
-    <Q1SevereSymptom key={0} onChange={onChange} />,
-    <Q2DifficultBreathing key={1} onChange={onChange} />,
-    <Q3Symptoms key={2} values={values} onChange={onChange} />,
-    <Q3SymptomBreathingSeverity key={3} values={values} onChange={onChange} />,
-    <Q3SymptomCoughSeverity key={4} values={values} onChange={onChange} />,
-    <Q3SymptomBodyAchesSeverity key={5} values={values} onChange={onChange} />,
-    <Q3SymptomSoreThroatSeverity key={6} values={values} onChange={onChange} />,
-    <Q4TestResult key={100} values={values} onChange={onChange} />,
-  ];
+  const decideNextPage = (values: Partial<SymptomCheckerForm>) => {
```
Comment (hitankar-freshworks) [inline apps/web/src/forms/SymptomChecker.tsx:61]: do we really want routes for each outcome? Not sure how it will be useful

Comment (vyasworks) [reply apps/web/src/forms/SymptomChecker.tsx:61]: I have updated based on existing things, we can change this conditional renders  once the happy path is done.

File: apps/web/src/components/Radio.tsx (hunk 3 above)
Comment (vyasworks) [reply apps/web/src/components/Radio.tsx]: I am going to leave this for now and address this change in an upcoming PR, to focus more on the flow now

**Pull request closed without merging**